client/me/profile-gravatar/index.jsx

```diff
@@ -12,80 +11,36 @@ import { connect } from 'react-redux';
 import Animate from 'components/animate';
 import Gravatar from 'components/gravatar';
 import { recordGoogleEvent } from 'state/analytics/actions';
-import { isEnabled } from 'config';
```

Comment:
I was going to suggest something similar. It can be helpful to leave the config flag for future issues.